interfaces/builtin/network_control.go

```diff
@@ -209,6 +209,9 @@ network sna,
 # For reading the address of a particular ethernet interface
 /sys/devices/{pci[0-9a-f]*,platform,virtual}/**/net/*/address r,
 
+# Some specific drivers are configured via config file
+/sys/devices/{pci[0-9a-f]*,platform,virtual}/**/config rw,
```

Comment:
That gives pretty broad access, this is not limited to the configuration of network device afaict? And if I read the kernel sysfs/driver code correctly this is a generic mechanism to expose configuration all sorts of devices from userspace? Do you know what specific devices are affected by this?